### ISSUE TITLE
odgi flip error

### ISSUE THREAD

**Issue by yuliamostovoy** [pangenome/odgi#433]
I'm getting an error when running "odgi flip" using an .og subgraph file built from an HPRC PGGB-generated graph from [here](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=pangenomes/freeze/freeze1/pggb/chroms/). My commands look like this:

```
odgi build -O -P -t 16 -g chr1.hprc-v1.0-pggb.gfa -o chr1.hprc-v1.0-pggb.og
odgi extract -i chr1.hprc-v1.0-pggb.og -o subgraph.hprc-v1.0-pggb.og -E -L 10000  -r grch38#chr1:1449689-1534685 -t 16 -P
odgi flip -i subgraph.hprc-v1.0-pggb.og -o subgraph.hprc-v1.0-pggb.flip.og -t 16

odgi: /home/jupyter/odgi/src/odgi.cpp:1649: virtual void odgi::graph_t::serialize_members(std::ostream&) const: Assertion `j == _path_count' failed.
Aborted (core dumped)
```
I'd appreciate any help, thanks!

**Comment by yuliamostovoy** [comment]
I got this to work by using odgi view to convert the subgraph to gfa, and then inputting that gfa to odgi flip. It just fails with the og file as input. A little downstream, I had the same thing happen with "odgi inject" - threw an error with the og file, worked fine if I converted the og to gfa first. Not sure if I'm doing something wrong here!

**Comment by ekg** [comment]
This is definitely a known issue. There is a solution that's easy which
I'll implement tomorrow (just copying the graph).

**Comment by AndreaGuarracino** [comment]
Fixed in #436.
